amazon/ionbenchmark/ion_benchmark_cli.py

```diff
@@ -318,8 +578,7 @@ def insert_into_report_table(table, row):
 def identify_report_table(command):
     if command == 'read':
         return identify_report_table_helper(
-            ['file_size (MB)', 'options', 'total_time (s)', 'conversion_\ntime (s)', 'conversion_time/\ntotal_time (%)',
-             'memory_usage_peak (MB)'])
+            ['file_size (MB)', 'options', 'total_time (s)', 'memory_usage_peak (MB)'])
```

Comment:
Yep, we should unify the read/write tables ultimately. I left it separately for now because Read/Write command may generate different metrics. 

One example is the previous `READ` table, which includes an additional `conversion time` column to indicate the execution time of whether raw Python objects are wrapped into simpleIon types.

Reply:
Thanks for explaining.